src/appComponents/tracks/TracksController.js

```diff
@@ -1,7 +1,9 @@
 /**
  * Created by championswimmer on 29/5/15.
  */
-/* exported error */
+
+(function() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function.](https://www.codacy.com/app/triplez/open-event-webapp/file/2431821597/issues/source?bid=3233857&fileBranchId=3238420#l5)